upstream_utils/libuv_patches/0010-Remove-swearing.patch

```diff
@@ -0,0 +1,54 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Jade Turner <spacey-sooty@proton.me>
+Date: Tue, 25 Jun 2024 13:10:31 +0800
+Subject: [PATCH 10/10] Remove swearing
+
+---
+ src/win/fs.c  | 6 +++---
+ src/win/tty.c | 2 +-
+ 2 files changed, 4 insertions(+), 4 deletions(-)
+
+diff --git a/src/win/fs.c b/src/win/fs.c
+index 26ba539936786902af5830c2811594292815d2d7..5796f10505965c7658ed37fae4fd2658ef12cddd 100644
+--- a/src/win/fs.c
++++ b/src/win/fs.c
+@@ -1185,7 +1185,7 @@ void fs__mktemp(uv_fs_t* req, uv__fs_mktemp_func func) {
+   size_t len;
+   uint64_t v;
+   char* path;
+-  
++
+   path = (char*)req->path;
+   len = wcslen(req->file.pathw);
+   ep = req->file.pathw + len;
+@@ -1658,7 +1658,7 @@ INLINE static int fs__stat_handle(HANDLE handle, uv_stat_t* statbuf,
+     statbuf->st_mode |= (_S_IREAD | _S_IWRITE) | ((_S_IREAD | _S_IWRITE) >> 3) |
+                         ((_S_IREAD | _S_IWRITE) >> 6);
+     statbuf->st_nlink = 1;
+-    statbuf->st_blksize = 4096;    
++    statbuf->st_blksize = 4096;
```

Comment:
There seems to be a lot of unrelated changes in this patch.  If there’s an update that needs to be done to sync with upstream, it shouldn’t be patched here.

Reply:
whoops thats my editor trimming whitespace :laughing: